css/rnx.css

```
@@ -0,0 +1,366 @@
+  font-family: var(--rnx-font-family-display);
+  letter-spacing: -0.02em;
+}
+
+code, pre, kbd, samp, .rnx-mono { font-family: var(--rnx-font-family-monospace); }
+
+/* 2b. per-component colour */
+
+.btn {
+  --bs-btn-border-radius: var(--rnx-border-radius-sm);
+  --bs-btn-font-weight: 500;
+  transition: var(--rnx-transition-fast);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)